docs/source/specs/openapi.yml

```diff
@@ -729,6 +771,105 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /api/v1/reports/costs/:
+    get:
+      summary: Query to obtain cost reports
+      operationId: getCostReports
+      tags:
+        - Report
+      parameters:
+        - in: header
+          name: Authorization
+          description: e.g. Token 45138a913da44ab89532bab0352ef84b
+          schema:
+            type: string
+          required: true
+        - in: query
+          name: filter
+          schema:
+            $ref: '#/components/schemas/ReportFilter'
+          description: The filter to apply to the report.
```

Comment:
Not a blocker, but beyond `filter`, we'll likely also need `group_by` and `order`.

Reply:
I'll pull `group_by` out of `filter`. What did you have in mind in terms of `order`? more than just `ascending` vs. `descending`?

Reply:
Yeah, ASC and DESC are fine.  

The short version is that in today's conversation about what the UI/UX will want to do, it'd be useful to be able to specify as part of the API query so that the response doesn't require additional re-sorting within the UI before it's displayed to the user.

So, the use case is, for example: `filter by ($service == "RDS"), group_by ($account && $region), order_by $cost DESC`